addons/mail/models/mail_thread.py

```diff
@@ -4511,20 +4511,20 @@ def message_get_followers(self, after=None, limit=100, filter_recipients=False):
 
     def _message_followers_to_store(self, store: Store, after=None, limit=100, filter_recipients=False, reset=False):
         self.ensure_one()
-        domain = [
+        domain = Domain([
             ("res_id", "=", self.id),
             ("res_model", "=", self._name),
             ("partner_id", "!=", self.env.user.partner_id.id),
-        ]
+        ])
         if filter_recipients:
             subtype_id = self.env["ir.model.data"]._xmlid_to_res_id("mail.mt_comment")
-            subtype_domain = [
+            subtype_domain = Domain([
```

Comment:
separate variable probably no longer necessary now that the syntax is easier

Reply:
I will simplify. It was just to keep the meaning.